routes/users.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.get('/', (req, res) => {
+  res.send('smoking tests');
+});
+
+router.get('/:user_id', (req, res) => {
+  knex.raw('SELECT * FROM users WHERE id = ?', [req.params.user_id]).then((result) => {
+    res.send(result.rows);
+  });
+});
+
+router.post('/login', (req, res) => {
+  knex
+    .raw('SELECT * FROM users WHERE email=?', [req.body.email])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        throw { message: 'User not found' };
+      } else if (result.rows[0].password !== req.body.password) {
+        throw { message: 'Incorrect Password' };
+      } else {
+        res.send(result.rows);
```

Comment:
Instead of sending an array of just one object, it's best to just return the object itself.